shared/attributes.asciidoc

```diff
@@ -56,6 +56,9 @@
 :plugins:              https://www.elastic.co/guide/en/elasticsearch/plugins/{branch}
 :cloud:                https://www.elastic.co/guide/en/cloud/current
 :ess-trial:            https://www.elastic.co/cloud/elasticsearch-service/signup?baymax=docs-body&elektra=docs
+:ess-product:          https://www.elastic.co/cloud/elasticsearch-service?baymax=docs-body&elektra=docs
+:ess-console:          https://cloud.elastic.co?baymax=docs-body&elektra=docs
+:ess-baymax:           ?baymax=docs-body&elektra=docs
```

Comment:
I added this attribute, because when I found a number of places in our ESS docs that used variants of the pages in lines 59 & 60 that we otherwise can't track (e.g. `https://www.elastic.co/cloud/elasticsearch-service/pricing`).